src/components/footer/HaukiFooter.css

```diff
@@ -0,0 +1,7 @@
+.page-footer {
+  margin-top: auto;
+}
+
+.hauki-koros pattern {
+  fill: #6feed3;
```

Comment:
Should we have some specific place for these theme-specific values?

Reply:
Maybe some theme.css or variables.css?

Reply:
Yes good idea!

- [ ] Separate file for theme-specific css variables